package.json

```diff
@@ -43,20 +43,24 @@
     "codegen:subgraph-sokol": "yarn --cwd packages/cardpay-subgraph codegen-sokol",
     "codegen:subgraph-xdai": "yarn --cwd packages/cardpay-subgraph codegen-xdai",
     "build:hub": "yarn --cwd packages/hub build",
+    "build:ember-shared": "yarn --cwd packages/ember-shared/addon build",
+    "build_if_missing:ember-shared": "yarn --cwd packages/ember-shared/addon build_if_missing",
```

Comment:
An alternative to this would be to put

```
"scripts": {
  "prepare": "build-if-missing"
}
```

in `packages/ember-shared/addon/package.json`.

This will do the build the first time you `yarn install` the monorepo.

Reply:
Ah, that seems much better. Thanks.

Reply:
@ef4 "prepare" didn't work but "postinstall" does. I did see a bit int he yarn docs warning away from postinstall though, so I thought I'd check with you if you had something else in mind.